scope.go

```diff
@@ -744,7 +744,11 @@ func newCluster(c config.Cluster) (*cluster, error) {
 		clusterUsers[cu.Name] = newClusterUser(cu)
 	}
 
-	heartBeat := newHeartBeat(c.HeartBeat, c.ClusterUsers[0])
+	heartBeatClusterUser := c.ClusterUsers[0]
```

Comment:
Taking occasion of your change, I don't understand what's the motivation of using the first user defined in the list for cluster. I think it can create a great confusion, which we actually already experienced in Contentsquare. Let's assign by default a default user for heartbeats. Could you do that?

Reply:
It is common to disable a user _default_ in ClickHouse completely, so we cannot use _default_ for heartbeat unconditionally.
Could you elaborate on ' assign by default a default user for heartbeats'?
Do you suggest  we should introduce clusters/heartbeat/name (and probably clusters/heartbeat/password), and if it is not specified use 'deafult'?

Reply:
I didn't know default user is being disabled ^^ it's a valid point. I will create a separate issue to add a specific user for heartbeats and fallback to the default one if not provided